src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -403,6 +403,29 @@ Tests:
       a: {^RandomInt: {distribution: poisson}}
     ThenThrows: InvalidValueGeneratorSyntax
 
+  - Name: Zipfian distribution
+    GivenTemplate:
+      a: {^RandomInt: {distribution: zipfian, alpha: 1.0, n: 10000}}
+    ThenReturns:
+      - a: { "$numberLong" : "3" }
```

Comment:
Suggestion / Nit: Maybe it is just me seeing patterns and jumping to conclusions but could you add more ThenReturns so that this looks more like a zipfian. Maybe 10 results in total.